views-jstachio/src/test/java/io/micronaut/views/jstachio/pkginfo/HomeModel.java

```diff
@@ -0,0 +1,10 @@
+package io.micronaut.views.jstachio.pkginfo;
+
+import io.jstach.jstache.JStache;
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.core.annotation.Nullable;
+
+@Introspected
+@JStache(path = "home")
```

Comment:
I think what you might not know is JStachio knows which classes have `JStache` annotations on them. The meta data is code generated. Thus you do not need `@Introspected`. Instead of asking micronaut what annotations are on this class and then go looking for JStache  you just ask JStachio does this class have a *compiled* template.

Reply:
[I have removed @Introspected annotation`](https://github.com/micronaut-projects/micronaut-views/pull/590/commits/62f63246086e35b2fdeb9eb29985a8ade7fac6be)